searchcore/src/tests/proton/documentdb/feedview/feedview_test.cpp

```diff
@@ -620,7 +622,8 @@ struct FixtureBase
     void removeAndWait(const DocumentContext &docCtx) {
         FeedTokenContext token(_tracer);
         RemoveOperationWithDocId op(docCtx.bid, docCtx.ts, docCtx.doc->getId());
-        runInMaster([&] () { performRemove(token.ft, op); });
+        runInMaster([this, ft=std::move(token.ft), &op] () { performRemove(std::move(ft), op); });
```

Comment:
Consider adding mutable specifier to the lambda expression.

Reply:
Fixed